src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -535,7 +535,11 @@ public void Initialize()
             StorageEnvironmentOptions options;
             if (Configuration.Core.RunInMemory)
             {
-                options = StorageEnvironmentOptions.CreateMemoryOnly();
+                options = StorageEnvironmentOptions.CreateMemoryOnly(                    
```

Comment:
Why we need this?

Reply:
Otherwise all nodes in the in-memory cluster will use the same dir, which might be a problem when deleting.

Reply:
reverted